src/coreComponents/functions/TableFunction.cpp

```diff
@@ -196,6 +196,11 @@ real64 TableFunction::evaluate( real64 const * const input ) const
   return m_kernelWrapper.compute( input );
 }
 
+std::vector< real64 > TableFunction::getCoord( real64 const * const input, InterpolationType interpolationMethod ) const
```

Comment:
@CusiniM i know std should not be used here, but not sure what should be the right container?

Reply:
Isn't it simply `array1d`?

Reply:
followed Dick's suggestion and added `dim` as argument, so return is simply `real64`